### PR TITLE
Update PR request template with review and testing notes

### PR DESCRIPTION
**Pull request by oskarth**
Update PR request template with review and testing notes

Also my first verified commit!

Per discussion in Slack to avoid regressions where upgrade path is ignored.

Review notes: Would you fill this in? Does it make sense?

Testing notes: Nothing to test.

status: ready

### CODE REVIEW

**Review by janherich — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -17,6 +17,12 @@ addresses #...
 - ...
 - Step 3, etc.
 
+[comment]: # (Specify if something in particular should be looked at, or ignored, during review)
+Review notes (optional):
+
+[comment]: # (Specify if something specific has to be tested, for example upgrade paths)
```
Comment (jeluard) [inline .github/PULL_REQUEST_TEMPLATE.md]: Could we have those comment as md comments (similar to what has been done in `ideas` rep). This way I don't have to remove it :)

**Review by jeluard — COMMENTED** [review]
Could we have those comment as md comments (similar to what has been done in ideas rep). This way I don't have to remove it :)

**Comment by oskarth** [comment]
@jeluard sure! something like this?

**Comment by jeluard** [comment]
@oskarth Actually I meant to have everything in comment so I can just let it if I don't modify it. But that may be too much :)

**Comment by oskarth** [comment]
@jeluard I think it is good if it is a bit in your face. But we might be able to simplify this in general. For example 'steps to test' only applies to what, 25% of PRs?

Maybe we can try this for a bit and see how it feels? Of course feel free to modify it to something else if you have a specific vision for how it should work.

**Review by jeluard — APPROVED** [review]

**Pull request merged**